src/drivers/x11/lv_x11.h

```diff
@@ -42,8 +42,8 @@ typedef void(*lv_x11_close_cb)(void * user_data);
  * create and add keyboard, mouse and scrollwheel objects and connect them to x11 display.
  *
  * This is a convenience method handling the typical input initialisation of an X11 window:
- * - create keyboard (@ref lv_x11_keyboard_create)
- * - create mouse (with scrollwheel, @ref lv_x11_mouse_create @ref lv_x11_mousewheel_create)
```

Comment:
What is the correct way to mark function in generated doc? So we can click it to jump to the function doc.

Reply:
@XuNeo @kdschlosser 

Good question!  Actually, Doxygen handles this automatically (adding the link automatically) by leaving the function name _bare_ with parens at the end.  Just make sure it is spelled the same as the actual function in source code, like this:

```
 * - create keyboard (lv_x11_keyboard_create())
 * - create mouse (with scrollwheel, lv_x11_mouse_create() and lv_x11_mousewheel_create())
```

--Vic

Reply:
@vwheeler63 

The answer is, it depends., It depends on what you are wanting to link to. If it is something that resides in the doxygen domain then it will be preserved. you cannot link to a specific place in the documentation that resides in one of the ReST files. You also want be able to link to a specific header file, this is because the documentation representation of the header file resides in the sphinx domain.

In sphinx (ReST) I am able to make a connection to the doxygen domain by using directives along the lines of 
```rest
:c:macro:`SOME_MACRO_NAME`
```

and I am able to reference specific header files by using

```rest
:ref:`header_name_h`
```

and I am able to link to sections of the documentation like the quick overview by using

```rest
:ref:`quick-overview`
```

There is no way to link to the quick overview from a doxygen docstring. YET... I am working on being able to use ReST syntax in the doxygen docstrings. Having this ability would help out a lot.

Reply:
>  The answer is, it depends.

Nice.

Doxygen will link to a file by just naming it _bare_ in a Doxygen comment.

Doxygen will also allow you to link to separate places within the Doxygen "realm", like if you create a link (several ways to do that, including the `@anchor` command) -- you can reference anywhere inside the same Doxygen "realm" by

```
@ref  my_anchor_name
```

and it will go there (in any file that was part of that Doxygen run).

But for sure (agreed) it's not going to know anything about any anchors in .RST files.